### ISSUE TITLE
About the post-processing implementation

### ISSUE THREAD

**Issue by s4k10503** [keijiro/YoloV4TinyBarracuda#11]
# Overview

Hi,

I would like to modify Postprocess1.compute and Postprocess2.compute like the reference repository (https://github.com/bubbliiiing/yolov4-tiny-keras) and the general implementation method. 
Additionally, I would like to add some features. 

So I'm considering creating a pull request. I have committed all my changes to my forked repository and am ready to submit a pull request. Below is a summary of the changes. Does this repository accept pull requests? 

Thank you.

### Modifications

- Change variable names to more descriptive ones.
- Change the method of calculating scores to the product of confidence of the box and the predicted probability of the class: sigmoid function shall be applied to each.

### Modifications

- Change variable names to more descriptive ones.
- Ensure that it is executed for each class.
- Sort the bounding boxes in order of scores before executing NMS.

### Added functionalities

- Implementation of Soft-NMS, an evolution of NMS. 
- Allow selection between NMS and Soft-NMS to be executed.

https://openaccess.thecvf.com/content_ICCV_2017/papers/Bodla_Soft-NMS_--_Improving_ICCV_2017_paper.pdf

## Miscellaneous

According to the changes in the above files, some additional files need to be modified as well. We will enable changing hyperparameters related to NMS from the Inspector in the Scene.
